packages/announcements/craco.config.js

```diff
@@ -0,0 +1,16 @@
+const cracoModuleFederation = require("craco-module-federation");
+const ExternalTemplateRemotesPlugin = require("external-remotes-plugin");
+
+module.exports = {
+  devServer: {
+    port: 3008,
```

Comment:
can you please check if this port does not conflicts with port in other packages?

Reply:
Checked, changed port to **4002**

Reply:
keep the pot number in 3000 series only